test/extended/setup.sh

```diff
@@ -15,9 +15,13 @@ function os::test::extended::focus () {
 	if [[ -n "${FOCUS:-}" ]]; then
 		exitstatus=0
 
+		local skip="\[Serial\]"
+		if [[ -n "${SKIP:-}" ]]; then
+		    skip+="|${SKIP}"
+		fi
```

Comment:
This seems to imply there was a bug of some sort?

Reply:
Yes, without this fix, ginko command will be:
```
ginkgo -v -noColor -focus=DNS -skip=\[Disabled:.+\]|\[Disruptive\]|\[Skipped\]|\[Slow\]|\[Flaky\]|\[local\]|\[Local\] -p -nodes=5  os::util::find::built_binary extended.test  -- -ginkgo.skip \[Serial\] -test.timeout 6h
```
Later ginkgo.skip option overrides the former -skip option and we do see execution of wildcard dns queries that has `[Disabled:Broken]` tag.